### ISSUE TITLE
Differentiate color-wise for closed, merged and open issues on project description popup

### ISSUE THREAD

**Issue by nkprince007** [coala/projects#298]
The popups for projects shows issues on the left side, but does not differentiate them as merged, closed or open. It'd be better if we had a real-time differentiation for the issues.

Using gitlab and github api to find details about issues would solve the issue.

**Comment by nkprince007** [comment]
CC @hemangsk

**Comment by KVGarg** [comment]
Good Idea! But won't that make GITHUB/GITLAB API calls, which is only limited to 60 for non-authenticated users.
